index.html

```diff
@@ -25,6 +25,20 @@
 		<div class="col-xs-4">
 			<iframe width="420" height="315" src="//www.youtube.com/embed/cOmC14DGddU?rel=0" frameborder="0" allowfullscreen></iframe>
 		</div>
+		
+		<div class="row col-xs-10 col-xs-offset-1">
+			<div>
+				<img src="./img/quote-open.png" align="left"/>
+			</div>
+			<div>
+				<img src="./img/quote-close.png" align="right"/>
+			</div>
+			<br />
+			<div>
+				<p  id="testimonial-row">
+				</p>
+			</div>
+		</div>
```

Comment:
In future, we should standardize 2 spaces as indentation for html files (because they contain lot of nesting levels).